### ISSUE TITLE
exported: not check all comment for `var` and `const`

### ISSUE THREAD

**Issue by mirecl** [mgechev/revive#1364]
**Describe the bug**
Not check comment line for  for `var` and `const` 

**Example code**
```go
package example

const Const = "" // Const ...
var Var = ""     // Var ...
```

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@1.3.4`
2. I run it with the following flags & configuration file:
3. Run
```shell
revive ./...
```
4. All good ✅ (not errors)
5. I updated revive `go install github.com/mgechev/revive@1.9.0`
6. Run
```shell
revive ./...
```
7. Errors ⛔
```shell
example.go:3:7: exported const Const should have comment or be unexported
example.go:4:5: exported var Var should have comment or be unexported
```

**Expected behavior**
All good ✅ (not errors)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version of Go: 1.23

**Comment by alexandear** [comment]
Thanks for the report!

This is actually expected behavior - not a bug.

In Go, comments for exported identifiers like `var` and `const` should be placed **above** the declaration, not to the right of it. Inline comments (to the right) are not considered documentation comments, and even [pkg.go.dev](https://pkg.go.dev) doesn't treat them as such - they won't show up in generated documentation.

<details><summary>Running pkgsite locally</summary>
<p>


```go
package example

const Const = "" // Const ...

var Var = "" // Var ...

// Const ...
const ConstValid = ""

// VarValid ...
var VarValid = ""
```

```console
$ pkgsite .

2025/06/23 13:17:18 Info: go/packages.Load(["all"]) loaded 257 packages from . in 113.618167ms
2025/06/23 13:17:18 Info: go/packages.Load(std) loaded 344 packages from /opt/homebrew/Cellar/go/1.24.4/libexec in 295.7145ms
2025/06/23 13:17:18 Info: FetchDataSource: fetching github.com/mgechev/revive@v0.0.0
2025/06/23 13:17:18 Info: FetchDataSource: fetching std@latest
2025/06/23 13:17:18 Info: Listening on addr http://localhost:8080
2025/06/23 13:17:18 Info: FetchDataSource: fetched std@latest using *fetch.goPackagesModuleGetter in 189.968084ms with error <nil>
2025/06/23 13:17:18 Info: FetchDataSource: fetched github.com/mgechev/revive@v0.0.0 using *fetch.goPackagesModuleGetter in 481.608875ms with error <nil>
```

<img width="1289" alt="Image" src="https://github.com/user-attachments/assets/422f0dbe-be07-4321-b85c-0c1f49e19919" />

</p>
</details> 



So to fix the warnings, update your code like this:

```go
// Const is ...
const Const = ""

// Var is ...
var Var = ""
```

Revive v1.9.0 likely improved the linter to follow this convention more strictly than v1.3.4.
